src/offsetPath.js

```diff
@@ -50,6 +83,16 @@
     } else if (path !== null) {
       var pathInput = path[1];
       return {type: 'path', path: pathInput};
+    } else {
+      var basicShapes = ['inset', 'circle', 'ellipse', 'polygon'];
+      if (!isInArray(basicShapes, shapeType[0])) {
```

Comment:
I think shapeType can be null causing this to throw. There should be a test case for this condition.